### PR TITLE
Migration for account_voucher module

### PR DESCRIPTION
**Pull request by arthru**

### CODE REVIEW

**Comment by bwrsandman** [comment]
Just update `openerp/openupgrade/doc/source/modules70-80.rst` to reflect the coverage of this module.
You will need to rebase with upstream/master

```
git remote add upstream https://github.com/OpenUpgrade/OpenUpgrade.git
git pull --rebase upstream master
```

File: addons/account_voucher/migrations/8.0.1.0/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---Fields in module 'account_voucher'---
+account_voucher / account.bank.statement.line / voucher_id (many2one)         : DEL relation: account.voucher
+account_voucher / account.voucher          / active (boolean)              : DEL 
+# Just let it be removed
+
+account_voucher / account.voucher          / currency_id (many2one)        : now required, default = function
+# Nothing to do : the function is not stored
+
+account_voucher / account.voucher          / message_last_post (datetime)  : NEW 
+# Need to compute the value
```
Comment (bwrsandman) [inline addons/account_voucher/migrations/8.0.1.0/openupgrade_analysis_work.txt]: Can you change this comment to reflect that it is being computed by the migration script.
This reads like it's a TODO for the next developers.

File: addons/account_voucher/migrations/8.0.1.0/openupgrade_analysis_work.txt [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---Fields in module 'account_voucher'---
+account_voucher / account.bank.statement.line / voucher_id (many2one)         : DEL relation: account.voucher
+account_voucher / account.voucher          / active (boolean)              : DEL 
+# Just let it be removed
```
Comment (bwrsandman) [inline addons/account_voucher/migrations/8.0.1.0/openupgrade_analysis_work.txt]: Maybe conserve the data here with `rename_columns()`

**Comment by bwrsandman** [comment]
:+1: 
LGTM

**Comment by hbrunn** [comment]
:+1:

**Pull request merged**